### ISSUE TITLE
Bpmn: editing elements'content property doesn't update the view

### ISSUE THREAD

**Issue by nico-onmap** [clientIO/joint#692]
It seems that the rappid 2.1 release has a different behaviour when calling `prop()` to update a model's properties.

When calling this:

```js
cellView.model.prop('content', 'new content');
```

This would update the model's `content` attribute and also update the view with the new content. However, with Rappid 2.1, this doesn't seem to update the view correctly anymore.

To reproduce, open the [bpmn demo](http://resources.jointjs.com/demos/rappid/apps/BPMNEditor/index.html)

Select this element:

![bpmn](https://user-images.githubusercontent.com/26772786/31230264-d8f05dda-a9e3-11e7-90ab-c76a77238a3f.png)

And change its `content`: notice how the view isn't updated.

I am using the bpmn components and noticed my views were not being updated anymore when the content is updated. This problem didn't happen with previous version.

**Comment by kumilingus** [comment]
Hi Nico, I've identified the issue and create a fix. It's caused by a leftover `attrs` property on the model. When you look at `CellView.model.attrs` for `{ div: { html: 'leftover text' }}` this rule wins over `{ '.content': { 'new text' }}`.

The following can temporarily solve the issue.
```javascript
cellView.model.prop('div/html', null);
cellView.model.prop('content', 'new content');
```

The actual fix will be part of the next release `v2.2` (next week). If you'd like to patch it now please email me at roman@client.io.

**Comment by nico-onmap** [comment]
Thank you for the quick answer. The workaround doesn't seem to work though.

Typing this in the bpmn demo doesn't update the view:

```js
graph.getCell('b2797827-0135-4c4d-8958-4b8bcd7144e9').prop('div/html', null);
graph.getCell('b2797827-0135-4c4d-8958-4b8bcd7144e9').prop('content', 'new content');
```

Doesn't update the view (but you can see that the content property has been updated if you select the element).

**Comment by kumilingus** [comment]
The workaround is even more ugly for now :)
```javascript
graph.getCell('b2797827-0135-4c4d-8958-4b8bcd7144e9').removeAttr('div/html');
graph.getCell('b2797827-0135-4c4d-8958-4b8bcd7144e9').prop('content', 'new content');
// fix participants rendering
v = graph.getCell('b2797827-0135-4c4d-8958-4b8bcd7144e9').findView(paper);
v.participantsAttrs = undefined;
v.update();
```

**Comment by kumilingus** [comment]
Fixed in Rappid `v2.2.0`.
